.gitlab-ci.yml

```diff
@@ -209,6 +209,7 @@ Test_N_Build:
   only:
   - dev
   - master
+  - new-cluster-deployments
```

Comment:
Can you change this to based on rules 

https://github.com/Ensembl/ensembl-web-metadata-api/blob/main/.gitlab-ci.yml#L55-L57
```
rules:
    - if: '$CI_DEPLOY_FREEZE == null && $CI_COMMIT_BRANCH == "main"'
      when: on_success
```